specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2018-01-01-preview/servicebus-preview.json

```diff
@@ -1,1437 +0,0 @@
-{
-  "swagger": "2.0",
```

Comment:
>swagger [](start = 3, length = 7)

Was the deleted file split into all the files in the same directory?

Reply:
> > "swagger": "2.0",
> 
> Was the deleted file split into all the files in the same directory?
> 
> Refers to: specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json:2 in [792f037](https://github.com/Azure/azure-rest-api-specs/commit/792f037ddde8fc1b5280affc307c0306ebd70c00). [](commit_id = [792f037](https://github.com/Azure/azure-rest-api-specs/commit/792f037ddde8fc1b5280affc307c0306ebd70c00), deletion_comment = True)

yes, the deleted file is split in all the files in the same directory.